src/PrestaShopBundle/Resources/views/Admin/AdvancedParameters/system_information.html.twig

```diff
@@ -0,0 +1,280 @@
+{#**
+ * 2007-2017 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2017 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+{% extends 'PrestaShopBundle:Admin:layout.html.twig' %}
+
+{% block content %}
+{% if notHostMode %}
+    <script type="text/javascript">
+        $(document).ready(function()
+        {
+            $.ajax({
+                type: 'POST',
+                url: '{{ path("admin_system_information_check_files") }}',
+                data: {},
+                dataType: 'json',
+                success: function(json)
+                {
+                    var tab = {
+                        'missing': '{{ "Missing files"|trans({}, "Admin.Advparameters.Notification") }}',
+                        'updated': '{{ "Updated files"|trans({}, "Admin.Advparameters.Notification") }}'
+                    };
+
+                    if (json.missing.length || json.updated.length)
+                        $('#changedFiles').html('<div class="alert alert-warning">{{ "Changed/missing files have been detected."|trans({}, "Admin.Advparameters.Notification") }}</div>');
+                    else
+                        $('#changedFiles').html('<div class="alert alert-success">{{ "No change has been detected in your files."|trans({}, "Admin.Advparameters.Notification") }}</div>');
+
+                        $.each(tab, function(key, lang)
+                        {
+                            if (json[key].length)
+                            {
+                                var html = $('<ul>').attr('id', key+'_files');
+                                $(json[key]).each(function(key, file)
+                                {
+                                    html.append($('<li>').html(file))
+                                });
+                                $('#changedFiles')
+                                    .append($('<h4>').html(lang+' ('+json[key].length+')'))
+                                    .append(html);
+                            }
+                        });
+                    }
+            });
+        });
+    </script>
```

Comment:
We should try to avoid embedding javascript in templates. Please store injected wordings and/or values as json in a javascript source, then move the script into a separate js file:

```twig
<script>
var input = {
  wording: {
    blahBlah: {{ 'blah blah'|json_encode }},
    blahBleh: {{ 'blah blah'|json_encode }},
  },
  foo: {{ someVar|json_encode }}
};
</script>
<script src="/path/to/script.js"></script>
```

So in `script.js` you simply consume the `input` variable.

If the script is simple, it would be even better to avoid the global variable altogether:

**script.js**
```js
var systemInformationPage = function systemInformationPage(input) {
  // do everything here
}
```

**template.twig**
```twig
<script src="path/to/script.js"></script>
<script>
systemInformationPage({
  wording: {
    blahBlah: {{ 'blah blah'|json_encode }},
    blahBleh: {{ 'blah blah'|json_encode }},
  },
  foo: {{ someVar|json_encode }}
});
</script>
```

Reply:
Ok

Reply:
I'd say `admin-dev/themes/new-theme/js/system-information-page/`